src/utils/testHelpers.js

```diff
@@ -83,7 +84,24 @@ export const mountWithRouterAndStore = (Component, props, routeConfig = {}, stor
 );
 
 /**
- * Mounts components that are wrapped in WithRouter
+ * Mounts components that are wrapped in Query Client
+ *
+ * @param {Class|Function} Component - A React component to be tested
+ * @param {Object} props - Set of props to be passed to the component
+ *
+ * @returns {Object} Mounted component
+ */
+export const mountWithQueryClient = (Component, props = {}) => {
```

Comment:
we have queryWrapper somewhere else reuse that one

Reply:
@soroushm Already reused. I'll remove this one